src/mngt/ocf_mngt_io_class.c

```diff
@@ -328,3 +329,30 @@ int ocf_mngt_cache_io_classes_configure(ocf_cache_t cache,
 
 	return result;
 }
+
+int ocf_repart_all_to_default(ocf_cache_t cache,
+	const struct ocf_mngt_io_classes_config *cfg)
+{
+	int i;
+
+	/* Temporarily disable IO classes (except for default) for
+	 * repartitioning.
+	 */
+	for (i = 1; i < OCF_USER_IO_CLASS_MAX; i++)
+		_ocf_mngt_set_partition_size(cache, i, 0, 0);
+
+	for (i = 1; i < OCF_USER_IO_CLASS_MAX; i++) {
+		ocf_lru_repart_all(cache,
```

Comment:
Why do we need to repart all ioclasses?

Reply:
The 3rd commit in this PR adds an optional parameter that decides whether only non-active or all partitions should undergo the process.